src/helpers.ts

```diff
@@ -1,17 +0,0 @@
-import { SubscriptionClient } from './client';
-import { print } from 'graphql/language/printer';
-
-// Quick way to add the subscribe and unsubscribe functions to the network interface
-export function addGraphQLSubscriptions(networkInterface: any, wsClient: SubscriptionClient): any {
```

Comment:
i wouldn't just remove it, at least at the begining, maybe add deprecation warning first for couple of versions.